docs/software/home_automation.md

```diff
@@ -183,4 +184,42 @@ Also installs:
 Source code: <https://github.com/reloxx13/TasmoAdmin>  
 Credits: Implemented by @svh1985
 
+## openHAB
+
+openHAB (open Home Automation Bus) is an open source home automation platform. The *keywords* are *Bindings*, *Things*, *Channels*, *Items*, *Rules*, *Pages*.
+
+![openHAB web interface screenshot](../assets/images/dietpi-software-homeautomation-openhab.jpg){: width="600" height="393" loading="lazy"}
+
+=== "Access to the web interface"
+
+    The web interface is accessible via port **8444**:
+
+    - HTTPS: `https://<your.IP>:8444`
+    - Username: `root`
+    - Password = `<your global password>` (default: `dietpi`)
```

Comment:
Those are not set on a fresh install but on first web UI access one can choose username and password of the initial admin acccount.
```suggestion
```

Reply:
Changed with the following push.